pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -456,7 +555,7 @@ func getLatestConfig(url string, iteration int,
 	}
 	saveReceivedProtoMessage(contents)
 
-	return inhaleDeviceConfig(config, getconfigCtx, false)
+	return inhaleDeviceConfig(config, fromController, getconfigCtx)
```

Comment:
Can we make inhaleDeviceConfig and parseConfig have the same order of arguments? In your PR arg2 and 3 are different for the two functions.

Reply:
OK, I have put context first, which is what we typically do. I have also fixed this for some other `parse*` methods in zedagent to make it unified.